packages/lens/src/validate.ts

```diff
@@ -1,13 +1,15 @@
 import {
   LensClient,
-  production,
-  type SimpleCollectOpenActionSettingsFragment,
   type MultirecipientFeeCollectOpenActionSettingsFragment,
+  type SimpleCollectOpenActionSettingsFragment,
+  production,
 } from '@lens-protocol/client'
-import { getClient } from './client'
-import { Chains } from '@rabbitholegg/questdk-plugin-utils'
+import axios from 'axios'
+import dotenv from 'dotenv'
```

Comment:
Just use `process.env` directly [as done here](https://github.com/rabbitholegg/questdk-plugins/blob/8e57c6e6844943058a072faf5a633f0f777339a5/packages/neynar/src/Neynar.ts#L30) and see if that works.

Reply:
Tried using process.env directly and it did not work. It logs as undefined.

But obviously it has worked for neynar package, so Im a bit confused 🫤

Reply:
I think it worked in Neynar because there was a check for `process.env`
https://github.com/rabbitholegg/questdk-plugins/blob/8e57c6e6844943058a072faf5a633f0f777339a5/packages/neynar/src/Neynar.ts#L43